CONTRIBUTING.md

```diff
@@ -82,3 +83,10 @@ And get the latest data:
 ``` bash
 $ make scrape_mlo_co2_since_2015
 ```
+
+# Start App
+
+```bash
```

Comment:
You can probably just use the Makefile target here for convenience? It's `make server`.

It shouldn't be necessary to pass any environment variables when you [configure the environment](https://github.com/eugeneotto/carbondoomsday/blob/1e142a79736e6e94bfb7ccc2c7cc371f1dda54c4/CONTRIBUTING.md#configure-the-environment).